### ISSUE TITLE
[Feature] Make SCP mobs

### ISSUE THREAD

**Issue by Maxattacker** [RussStation/RussStation#35]
As a base, make SCP into basic mobs. Pretty much copy the human mob code and have them set to the corresponding sprites.

**Comment by Maxattacker** [comment]
Updating with deets.
I want to make SCP mobs simple animals that have antagonist objectives, similar to Revenant. 
Currently, I'd like this function to apply to SCP 049, SCP 106, and SCP 173. For those not familiar, the plague doctor, the old man, and the man-eating statue, respectively. I do have the sprites for all of these as well. Abilities to come later.
As for another small thing, I'd like 2 friendly SCP, being basic simplemobs, and that is SCP 529, and SCP 999. The half cat, and the gelatin.

**Comment by BobJoga** [comment]
Hippie has some code and sprites for SCP mobs. It might help in this case, might not.

**Comment by CPhantasm** [comment]
Are the sprites you mentioned still available?

**Comment by Maxattacker** [comment]
> 
> 
> Are the sprites you mentioned still available?

Unfortunately in the 2 years since this was posted, the hard drive these sprites were on stopped functioning. Sorry.

**Comment by CPhantasm** [comment]
Yeah I figured it was a longshot, thank you for the prompt response
